data/brands/amenity/cafe.json

```
@@ -3507,6 +3507,18 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "Puccino's",
+      "locationSet": {"include": ["gb", "ie"]},
```

Looking at their website, they only have locations in England and Ireland
```suggestion
      "locationSet": {"include": ["gb-eng", "ie"]},
```

---

Good call, done.